### PR TITLE
Fixes #9079: When a ncf technnique has an ampersand in its name, rudder doesn't see it

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9079

### CODE REVIEW

File: tools/ncf_rudder.py [hunk 1]
```diff
@@ -159,15 +159,22 @@ def write_rudder_reporting_file(path,technique):
   file.close()
 
 
+def escape_entities( str ):
+    str = str.replace("&", "&amp;")
+    str = str.replace("<", "&lt;")
+    str = str.replace(">", "&gt;")
+    str = str.replace("\"", "&quot;")
```
Comment (amousset) [inline tools/ncf_rudder.py]: don't we need single quote too?

Comment (peckpeck) [inline tools/ncf_rudder.py]: Attributes in xml should be put within double quote and that's what we do.

Comment (amousset) [inline tools/ncf_rudder.py]: It seems to be only a common convention (http://stackoverflow.com/questions/6800467/quotes-in-xml-single-or-double), but we indeed use only double quotes.

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**